source/docs/articles/local.md

```diff
@@ -6,7 +6,7 @@ category:
 - getting-started
 keywords: local development, pantheon, develop locally, developing
 ---
-While Pantheon provides several options for on-server development, [local development](/docs/articles/local/developing-locally) has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions; however, we can provide some suggestions and known working solutions. If you're looking for a self-contained local development solution, check out [Kalabox](http://www.kalamuna.com/products/kalabox), which integrates with the Pantheon platform.
+While Pantheon provides several options for on-server development, [local development](/docs/articles/local/developing-locally) has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions; however, we can provide some suggestions and known working solutions.
```

Comment:
This still feels like leftover content from the local setup doc, is anyone opposed to deleting the intro entirely here? The existing header, **Resources for Local Development**, is good enough for describing the subject of the listed docs IMO